LobbyHelper.cs

```diff
@@ -69,6 +70,21 @@ public static class LobbyHelper {
             return null;
         }
 
+        public static int GetLobbyIndex(string sid) {
+            if (!IsCollabLobby(sid)) return -1;
+
+            const string lobbies = "/0-Lobbies/";
+            var index = sid.IndexOf(lobbies, StringComparison.Ordinal);
+            if (index < 0) return -1;
+
+            sid = sid.Substring(index + lobbies.Length);
+
+            index = sid.IndexOf('-');
```

Comment:
`RelativeIndex` seems to be inconsistent, sometimes it gave me one off (since it counts `0-Prologue`), and other times all the lobbies would return 17???
Using `LevelSetStats.Areas.FindIndex()` for the lobby's SID (and accounting for prologue) seemed to be much better).